dev/user.clj

```diff
@@ -1,5 +1,6 @@
 (ns user
-  (:require [datahike.api :as d]))
+  (:require [datahike.api :as d]
+            [hashp.core]))
```

Comment:
I assume that the user needs to eval that manually.

Reply:
Added the user.clj library